ott.install

```diff
@@ -0,0 +1,3 @@
+bin: ["src/ott"]
+doc: ["doc/ott_manual_0.25.pdf" "doc/ott_manual_0.25.html"]
```

Comment:
the `doc/ott_manual_0.25.*` are no longer around, I adjusted it in eb88000 to use `built_doc/top2.*`